File: capt.rb

birthYear = 1979
birthMonth = 9
birthDay = 15
currentYear = 2016
currentMonth = 11
currentDay = 13
age = currentYear-birthYear

if currentMonth < birthMonth || currentMonth == birthMonth && currentDay < birthDay
    age = age-1
end
	
print "The captain is #{age} years old.\n"

File: capt.java
 public class capt{

    public static void main(){

int birthYear = 1979;
int birthMonth = 8;
int birthDay = 15;
int currentYear = 2016;
int currentMonth = 4;
int currentDay = 13;
int age = currentYear-birthYear;

if(currentMonth < birthMonth || currentMonth == birthMonth && currentDay < birthDay)
    age -= 1;

System.out.println("The captain is " + age +  " years old.");

    }

}
